src/views/AddExpenses.js

```diff
@@ -1,8 +1,12 @@
 import { useState } from 'react';
-import { convertEurosToCents, getUKFormattedDate } from '../utils/helpers';
+import { convertEurosToCents } from '../utils/helpers';
 import DatePicker from 'react-datepicker';
 
-const AddExpenses = ({ addExpense, setNewExpenseMonth }) => {
+import { useGlobalFunctions } from '../store/hooks';
+
+const AddExpenses = () => {
+  const { addExpense } = useGlobalFunctions();
```

Comment:
i am not sure why we need global functions at all.

if we removed the separation into functions and state, could i also write this?

```js
const { addExpense } = useContext(GlobalContext);
```

Reply:
Yes, that would work. If we run into a situation where we need to destructure the global context into a long list of variables and functions, the custom hooks `useGlobalState` and `useGlobalFunctions` would potentially improve readability. But maybe it's too early for this type of consideration.

Reply:
i would personally prefer the simplest/directest version possible.